systemd/suse-migration-reboot.service

```diff
@@ -1,6 +1,6 @@
 [Unit]
 Description=Reboot System
-After=suse-migration-kernel-load.service
+After=suse-migration-kernel-load.service suse-migration-update-bootloader.service
```

Comment:
This probably should have an accompanying "documentation" change in https://github.com/SUSE/suse-migration-services/blob/master/helper/service.tree

Reply:
This part was already in the "documentation", but not really enforced. I've updated service.tree file anyway with more services.